mysql-test/suite/galera/r/galera_bf_abort.result

```diff
@@ -4,7 +4,7 @@ START TRANSACTION;
 INSERT INTO t1 VALUES (1);
 INSERT INTO t1 VALUES (1);
 INSERT INTO t1 VALUES (2);
-ERROR 40001: Deadlock found when trying to get lock; try restarting transaction
+ERROR 40001: WSREP cancelled the transaction
```

Comment:
Well we can avoid to be specific (log should have the details) and keep it generic but with hint of what kind of failure it is about.

How about this message:

"WSREP detected deadlock or conflict. Transaction aborted. Trying restarting transaction"